components/blocks/nav/SiteNavbar/SiteNavbar.vue

```diff
@@ -28,69 +17,28 @@
               d="M12 8v4l3 3m6-3a9 9 0 11-18 0 9 9 0 0118 0z"
             />
           </svg>
-          <h1
-            class="
-              w-full
-              py-2
-              text-lg text-center
-              md:text-left
-              font-semibold
-              leading-6
-              tracking-tight
-              hidden
-              md:inline
-            "
-          >
-            speedrun.website
-          </h1>
+          <h1>leaderboards.gg</h1>
         </NuxtLink>
       </div>
-      <div
-        :class="{ hidden: !mobileNavIsActive }"
-        class="
-          lg:flex
-          border-t border-gray-300
-          lg:border-t-0
-          flex-col
-          lg:flex-row
-          mt-2
-          lg:mt-0
-          w-full
-          lg:w-auto
-          space-y-2
-          lg:space-y-0
-        "
-      >
-        <CoreNavLinks />
-        <CoreSearchBar />
+      <!--      mobile menu when click hamburger  -->
+      <div :class="{ hidden: !mobileNavIsActive }" class="mobile-menu">
+        <NavLinks />
+        <SearchBar />
       </div>
-      <div
-        class="
-          flex flex-row
-          items-center
-          absolute
-          right-2
-          top-4
-          lg:relative lg:right-0 lg:top-0
-          space-x-2
-          lg:space-x-0
-        "
-      >
-        <CoreLoginButton
+      <!--      mobile nav bar buttons-->
+      <div class="mobile-navbar">
+        <LogInButton
           data-testId="site-navbar-login-button"
           @click="toggleLoginModal"
           @keyup.enter="toggleLoginModal"
         />
-        <CoreSignUpButton
+        <SignUpButton
           data-testId="site-navbar-sign-up-button"
           @click="toggleSignUpModal"
           @keyup.enter="toggleSignUpModal"
         />
-
-        <button
-          class="items-center rounded text-black lg:hidden"
-          @click="toggleMenu"
-        >
+        <!--hamburger menu-->
+        <button class="mobile-menu__toggle" @click="toggleMenu">
```

Comment:
💭 If we need this comment still, maybe a more descriptive class name would be better?